### PR TITLE
Using latest version of terminal-notifier on OSX, closes getlantern/l…

### PR DESCRIPTION
**Pull request by oxtoacart**
…antern-internal#866

### CODE REVIEW

File: OSXNotifier.go [hunk 1]
```diff
@@ -44,5 +46,10 @@ func (n *osxNotifier) Notify(msg *Notification) error {
 		return err
 	}
 	log.Debugf("Received result: %v", string(result))
+	if msg.ClickURL != "" {
+		if string(result) == "Open" {
```
Comment (oxtoacart) [inline OSXNotifier.go:65]: This is the heart of the change. We specify our own action, which allows us to get the result of the click here, and then we just open the URL ourselves.

File: OSXNotifier.go [hunk 2]
```diff
@@ -30,12 +31,13 @@ func (n *osxNotifier) Notify(msg *Notification) error {
 	args := []string{
 		"-message", msg.Message,
 		"-title", msg.Title,
+		"-timeout", "15",
 	}
 	if msg.ClickURL != "" {
-		args = append(args, []string{"-open", msg.ClickURL}...)
+		args = append(args, "-actions", "Open")
```
Comment (oxtoacart) [inline OSXNotifier.go]: It occurs to me that we need to be able to localize this (i.e. pass it in).

File: OSXNotifier.go [hunk 3]
```diff
@@ -27,15 +30,29 @@ type osxNotifier struct {
...
 		"-message", msg.Message,
 		"-title", msg.Title,
+		"-timeout", fmt.Sprintf("%d", int(timeout.Seconds())),
+	}
+	if msg.Sender != "" {
+		args = append(args, "-sender", msg.Sender)
+		// override IconURL
+		msg.IconURL = ""
 	}
 	if msg.ClickURL != "" {
```
Comment (myleshorton) [inline OSXNotifier.go:47]: This should be msg.ClickLabel I think right @oxtoacart?

**Comment by myleshorton** [comment]
Nice @oxtoacart! Just one comment.

File: OSXNotifier.go (hunk 3 above)
Comment (oxtoacart) [reply OSXNotifier.go:47]: It's actually right. We only use ClickLabel if there's a ClickURL

Comment (myleshorton) [reply OSXNotifier.go:47]: OK gotcha. LGTM!

**Pull request merged**